lightning/src/ln/wire.rs

```diff
@@ -219,22 +231,32 @@ pub fn read<R: io::Read>(buffer: &mut R) -> Result<Message, msgs::DecodeError> {
 /// # Errors
 ///
 /// Returns an I/O error if the write could not be completed.
-pub fn write<M: Encode + Writeable, W: Writer>(message: &M, buffer: &mut W) -> Result<(), io::Error> {
-	M::TYPE.write(buffer)?;
+pub fn write<M: Type + Writeable, W: Writer>(message: &M, buffer: &mut W) -> Result<(), io::Error> {
```

Comment:
Does this need to remain `pub`?

Reply:
No put as `pub(crate)`.

Reply:
This doesn't appear to have changed in the latest PR commits?

Reply:
Oups sorry it got lost during rebase. Fixed